whois-nrtm4/src/main/java/net/ripe/db/nrtm4/persist/NrtmVersionDao.java

```diff
@@ -56,11 +57,12 @@ public void createNew(final NrtmSource source) {
         final long version = 1L;
         final UUID sessionID = UUID.randomUUID();
         jdbcTemplate.update(
-                "insert into version_information (source_id, version, session_id) " +
-                        "values (?, ?, ?)",
```

Comment:
Formatting. we use he convention of uppercase for reserved keywords like select, insert. It makes it more readable

Reply:
sure. i just like lower case for everything ;)

Reply:
I guess you have to start pressing few more keys till Intellij becomes smart enough ;)

Reply:
done in 'transform-rpsl-to-nrtm-json'